react-navigation-and-stack-navigator/readme.md

```diff
@@ -3,4 +3,55 @@
 ## Guide For
 
 > #### React-Navigation And Stack Navigator
-> Set up core navigation and create basic app navigation with stack navigator.
\ No newline at end of file
+> Set up core navigation and create basic app navigation with stack navigator.
+
+### Installation
+```
+npm install @react-navigation/native
+or 
+yarn add  @react-navigation/native
+```
+
+#### Installing dependencies into a bare React Native project
+```
+npm install react-native-reanimated react-native-gesture-handler react-native-screens react-native-safe-area-context @react-native-community/masked-view
+```
+If you're on a Mac and developing for iOS, you need to install the pods (via Cocoapods) to complete the linking.
+```
+npx pod-install ios
+```
+
+#### Installing Stack Navigator
+```
+npm install @react-navigation/stack
+or
+yarn add @react-navigation/stack
+```
+
+#### To finalize installation of react-native-gesture-handler, add the following at the top (make sure it's at the top and there's nothing else before it) of your entry file, such as index.js or App.js:
```

Comment:
untuk example nya aku lebih ke nambah contoh struktur folder dan file @nahansans di taruh di folder section fitur ini

Reply:
maksute kayak gimana kak?

Reply:
nambah file App.js @nahansans isinya konfigurasi navigasi dan stack juga nambah folder sources/screens isinya dua layar misal Splashscreen.js dan Home.js di folder fitur ini

Reply:
cukup itu aja ga perlu sampe package.json atau file dari init react-native ikut ke dalam folder ini

Reply:
aku gk bisa buat file baru disini kak, akses buat file nya gk bisa

Reply:
ketika kamu create file di push dan di pr dia ngga bisa ya?

Reply:
ya uda yg pr #1 dulu kurang dikit @nahansans setelah merge #1 aku pull di lokal lalu buat filenya

Reply:
oke

Reply:
> aku gk bisa buat file baru disini kak, akses buat file nya gk bisa

uda @nahansans disini empty folder dan file nya https://github.com/crocodication/guide_for_react_native_libraries/tree/master/react-navigation-and-stack-navigator